quickwit/quickwit-metastore/test-data/index-metadata/v0.5-97f3a089a54400241da1cab788695aca.expected.json

```diff
@@ -0,0 +1,109 @@
+{
+  "checkpoint": {
+    "kafka-source": {
+      "00000000000000000000": "00000000000000000042"
+    }
+  },
+  "create_timestamp": 1789,
+  "incarnation_id": "0001EHZADJ3VAF00JK8RQ0VMSA",
+  "index_config": {
```

Comment:
two "index-metadata" files where created.
One is sufficient (see PR comment). Ideally v0.6.

Reply:
I am not sure how to change it to v0.6 without migrating the protocol version, which I thought we decided to postpone until later in the release cycle.

Reply:
ok Iguess we can clean up those files later.